scripts/create-fuels-template-integration.sh

```diff
@@ -1,7 +1,64 @@
 #!/bin/bash
 
-(cd templates/nextjs && pnpm run fuels:dev) &
-cp templates/nextjs/.env.example templates/nextjs/.env.local
-(cd templates/nextjs && pnpm run dev) &
+# Debugging
+# PUBLISHED_NPM_VERSION="next"
+
+# Versions
+FUEL_CORE_VERSION="0.27.0"
+FORC_VERSION="0.60.0"
+TOOLCHAIN="CI"
+
+# Project
+PROJECT_DIR="test-project"
+
+pkill fuel-core
+
+echo "1. Install toolchains"
+if [ -x "$(command -v fuelup)" ]; then
+  echo "Fuelup exists"
+else
+  echo "Fuelup does not exist - installing 'fuelup'"
+  curl -fsSL https://install.fuel.network/ | sh -s -- --no-modify-path
+  export PATH="${HOME}/.fuelup/bin:${PATH}"
+fi
+
+fuelup toolchain new $TOOLCHAIN
+fuelup default $TOOLCHAIN
+fuelup component add fuel-core@$FUEL_CORE_VERSION
+fuelup component add forc@$FORC_VERSION
+
+echo "2. Scaffold a new project with 'create fuels@$PUBLISHED_NPM_VERSION'"
+if [ -d "$PROJECT_DIR" ]; then
+  echo "Removing existing project directory '$PROJECT_DIR'"
+  rm -rf $PROJECT_DIR
+fi
+pnpm create fuels@$PUBLISHED_NPM_VERSION $PROJECT_DIR --pnpm -cps --no-install
+
+echo "3. Intialise the project"
+cd $PROJECT_DIR
+pnpm add fuels@$PUBLISHED_NPM_VERSION > /dev/null 2>&1
+pnpm  --ignore-workspace install > /dev/null 2>&1
```

Comment:
Why do we have this?

Shouldn't it be automatic via `fuels@$PUBLISHED_NPM_VERSION`?

Reply:
Hmm, I think I'm just making many high-level assumptions here.

I'll probably have to review it again.